src/components/ebay-menu/test/test.browser.js

```diff
@@ -11,6 +11,56 @@ afterEach(cleanup);
 /** @type import("@marko/testing-library").RenderResult */
 let component;
 
+describe('typeahead functionality', () => {
+    const input = mock.Countries;
+    const firstItemText = input.items[0].renderBody.text;
+    const secondItemText = input.items[1].renderBody.text;
+    const thirdItemText = input.items[2].renderBody.text;
+
+    beforeEach(async() => {
+        component = await render(template, input);
+    });
+    describe('first', () => {
+        beforeEach(async() => {
+            await fireEvent.click(component.getByText(firstItemText));
+            await fireEvent.keyPress(component.getByText(firstItemText), { key: 'a', code: 65 });
+            await fireEvent.keyPress(component.getByText(firstItemText), { key: 'l', code: 76 });
+            await fireEvent.keyPress(component.getByText(firstItemText), { key: 'c', code: 67 });
+        });
+
+        it('shows the correct item in focus when the user types', async() => {
+            await waitFor(() => {
```

Comment:
Did you try to do this without the `waitFor`, not sure if it's needed. 
In the text below you have a `waitFor` and in there there are other `fireEvents` and an assertion without a `waitFor`